src/System.ComponentModel.TypeConverter/src/System/ComponentModel/ReflectTypeDescriptionProvider.cs

```diff
@@ -127,6 +128,12 @@ internal ReflectTypeDescriptionProvider()
             [typeof(Guid)] = typeof(GuidConverter),
             [typeof(Uri)] = typeof(UriTypeConverter),
             [typeof(Color)] = typeof(ColorConverter),
+#if netcoreapp
```

Comment:
You're just missing to define the constant through the csproj files. 

```xml
<DefineConstants Condition="'$(TargetGroup)'=='netcoreapp'">$(DefineConstants);netcoreapp</DefineConstants>
```

https://github.com/dotnet/corefx/blob/0b0f63d47951773cc46393312d80b40176d3a413/src/System.Reflection.DispatchProxy/tests/System.Reflection.DispatchProxy.Tests.csproj#L4

You'll need this in both, ref and src csproj.

Reply:
Thanks. Still learning something... 😃